custom/enikshay/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class IssuerId(models.Model):
+    """
+    This model is used to ensure unique, incrementing issuer IDs for users,
+    and to look up a user given an issuer ID.
+    obj.pk represents the serial issuer ID, later representations will be added as fields
```

Comment:
sounds fine if we use an alias for `issuer_id`. I don't really like the idea of accessing the `pk` to also mean 'issuer_id'.